_config.yml

```diff
@@ -18,6 +21,7 @@ plugins:
   - jekyll-redirect-from # for 301 redirects
   - jekyll-relative-links # to convert internal MD links to HTML
   - jekyll-optional-front-matter # for Markdown without front matter (e.g., CLI reference)
+  - jekyll-sitemap # produce a sitemap.xml file for submission to search engines
```

Comment:
We should exclude `sitemap.xml` from the search index by adding a check to `search-data.json`.

Reply:
Made this change